### PR TITLE
DOP-909: Convert docs-landing to Snooty project

### PR DESCRIPTION
**Pull request by mango-db**
[DOP-909](https://jira.mongodb.org/browse/DOP-909).

[Docs-landing staging](https://docs-mongodbcom-staging.corp.mongodb.com/landing/docsworker/DOP-909/legacy).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,50 +0,0 @@
-GIT_BRANCH=`git rev-parse --abbrev-ref HEAD`
```
Comment (sophstad) [inline Makefile:1]: We can't delete this file yet! Although we hope to move all of these landing pages over to the Snooty toolchain, for now we want to ensure that both Makefile targets _and_ Snooty work here. From the [spec](https://docs.google.com/document/d/1zgKt07n4wudcmwh8p-Vvqo3_4THeuUykCVmejgT5EtQ/edit#):
> ...the current workflow of the docs-landing repository will be uninterrupted; the Makefile targets will continue to function as they do now.

File: snooty.toml [hunk 2]
```diff
@@ -1,7 +1,16 @@
 name = "landing"
-title = "MongoDB"
+title = "MongoDB Documentation"
+default_domain = "landing"
 
 [constants]
+title = "MongoDB Documentation"
```
Comment (sophstad) [inline snooty.toml:6]: Title is specified above; no need to include it here!

The `[constants]` table specifically implements [source constants](https://docs.mongodb.com/meta/reference/source-constants/), which are referenced in documentation using `{+this+}` syntax. The `title` field has a different, specific purpose: populating `<title>` tags and the site's sidebar.

File: source/legacy.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+:template: landing
+
+====================
+Legacy Documentation
+====================
+
+MongoDB encourages upgrading to the most recent version to ensure documentation support. If upgrading is not possible, archived documentation is still available for previous versions. Archived documentation no longer receives updates.
```
Comment (sophstad) [inline source/legacy.txt:7]: MongoDB docs follow a [hard-wrap guideline of 72-80 characters per line](https://docs.mongodb.com/meta/practices/), and, unfortunately, this is implemented manually 😅 let's follow those guidelines and add some linebreaks!

File: source/legacy.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (sophstad) [inline source/legacy.txt:8]: Forgot to include this in the AC, but the [design](https://mongodb.invisionapp.com/share/H8W3OGJTRS9#/screens/406108043)  includes a horizontal line before the dropdowns. In docutils, this is referred to as a [transition marker](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#transitions) and is represented by dashes:
```
Para.

----------

Para.
```

File: source/legacy.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+.. depecated-version-selector::
```
Comment (sophstad) [inline source/legacy.txt:9]: Typo!
```suggestion
.. deprecated-version-selector::
```

**Comment by mango-db** [comment]
Docs-landing staging: [regular](https://docs-mongodbcom-staging.corp.mongodb.com/landing/docsworker/DOP-909/legacy) and [x-large](https://docs-mongodbcom-staging.corp.mongodb.com/landing/docsworker-xlarge/DOP-909/legacy).

**Pull request merged**